test/Microsoft.ML.Core.Tests/UnitTests/TestCSharpApi.cs

```diff
@@ -739,5 +739,64 @@ public void TestCrossValidationMacroWithNonDefaultNames()
                 }
             }
         }
+
+        [Fact]
+        public void TestOvaMacro()
```

Comment:
I see that this PR is for fixing normalization issue. However, this test does not test for normalization anywhere. Is this test specifically for normalization or its just the test for "OVA"?

Reply:
It test it through Accuracy! Without normalization Accuracy macro is only 0.6, and this is how I discover this bug.

Reply:
Initially I plan just add test for Ova, but metrics were really bad (around 0.6 accuracy) so I start digging and found problem with normalization. So indirectly it tests presence of normalization through accuracy metric.

---
In reply to: [193240523](https://github.com/dotnet/machinelearning/pull/310#discussion_r193240523) [](ancestors = 193240523)